modules/monitoring-configuring-metrics-collection-profiles.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring-metrics-collection-profiles_{context}"]
+= Configuring metrics collection profiles
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Metrics collection profiles are a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production. 
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+By default, Prometheus collects a large amount of time-series metrics data from default metrics targets.
+Using a metrics collection profile for core {product-title} monitoring components, you can choose to collect either the default amount of metrics data or a minimal amount of data. 
```

Comment:
Reading this to me seems a bit that by default we are collecting too many metrics. Let me know if by change you also feel the same or it's just me. 

Maybe I'm more on the side of not saying we collect a large number of metrics but rather that we collect all the metrics exposed by the default targes. However, in some cases:
- a clusters reach a big enough size where those default metrics demand users to run bigger prom instances
- users only care about the platform alerts and never look at the other metrics 

Is that we can discard some metrics, and should use a metric collection profile.

Reply:
That's a good point. I think I need to reword it.

Reply:
Much better IMO! Feel free to mark this as resolved